<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModelEditorClass</class>
 <widget class="QMainWindow" name="ModelEditorClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ModelEditor</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_F">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="openObjFile"/>
    <addaction name="openM3dFile"/>
    <addaction name="openMEProjFile"/>
    <addaction name="saveFile"/>
   </widget>
   <addaction name="menu_F"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_save"/>
   <addaction name="action_play"/>
   <addaction name="action_stop"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="openObjFile">
   <property name="text">
    <string>打开.obj文件</string>
   </property>
  </action>
  <action name="saveFile">
   <property name="text">
    <string>保存文件</string>
   </property>
  </action>
  <action name="openM3dFile">
   <property name="text">
    <string>打开.m3d文件</string>
   </property>
  </action>
  <action name="openMEProjFile">
   <property name="text">
    <string>打开.meproj文件</string>
   </property>
  </action>
  <action name="action_save">
   <property name="icon">
    <iconset resource="modeleditor.qrc">
     <normaloff>:/ModelEditor/Resources/Save.png</normaloff>:/ModelEditor/Resources/Save.png</iconset>
   </property>
   <property name="text">
    <string>保存xml文件</string>
   </property>
   <property name="toolTip">
    <string>保存文件</string>
   </property>
  </action>
  <action name="action_play">
   <property name="icon">
    <iconset resource="modeleditor.qrc">
     <normaloff>:/ModelEditor/Resources/Play.png</normaloff>:/ModelEditor/Resources/Play.png</iconset>
   </property>
   <property name="text">
    <string>播放</string>
   </property>
  </action>
  <action name="action_stop">
   <property name="icon">
    <iconset resource="modeleditor.qrc">
     <normaloff>:/ModelEditor/Resources/Stop.png</normaloff>:/ModelEditor/Resources/Stop.png</iconset>
   </property>
   <property name="text">
    <string>停止</string>
   </property>
   <property name="toolTip">
    <string>停止</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="modeleditor.qrc"/>
 </resources>
 <connections/>
</ui>
